picard/const/__init__.py

```diff
@@ -35,6 +35,7 @@
 )
 
 USER_PLUGIN_DIR = os.path.join(USER_DIR, "plugins")
+USER_DOWNLOADS_DIR = os.path.join(USER_DIR, "downloads")
```

Comment:
Network cache doesn't count, we don't want to interact directly with it anyway.

I'm not sure to understand, but i guess we could:
- download zipped plugin archive directly to user plugin directory
- load the plugin (zip) using zipimport when it is enabled

It will not change much from the user point of view, so i think we can implement this later (or sooner if you provide a patch).

I'm not against such move, but it is a slight improvement, disk space isn't much an issue, and code will not be much simpler as we still need to allow non-zipped plugin installation.

That said, i think we should get rid of this USER_DOWNLOADS_DIR anyway.